docs/source/configuration/environmentvariables.md

```diff
@@ -9,193 +9,190 @@ myst:
 
 # Environment variables
 
-All the environment variables defined at runtime that have the "RAZZLE_" prefix, are available in the browser under window.env
+This page describes environment variables and their usage for configuration of your Volto application at runtime.
 
-ex:
-If we start the application with an env variable
 
-```bash
+## Access environment variables
```

Comment:
This section is sort of an obscure debugging hint. It seems odd to put it above the actual list of environment variables, which is probably what someone reading this page is looking for.

Reply:
Although obscure, is it frequently useful? I assumed it was applicable all the time in the general sense.

If it is rarely useful for only a specific case, then this heading and text should reflect that _and_ be demoted.

Reference documentation usually flows from general usage for all, to specific usage for each, and then the special cases. Of course, there are always exceptions.

Reply:
@stevepiercy Well, I didn't know about it before, and I haven't missed it.

Reply:
Gotcha. Fixed in [1a639ab](https://github.com/plone/volto/pull/5736/commits/1a639ab55084b7ea67e5eddf6db16031acf8919b)